package.json

```diff
@@ -24,6 +24,7 @@
     "remark-cli": "~2",
     "remark-lint": "~5",
     "remark-validate-links": "~5",
+    "sass-lint": "~1.12.1",
```

Comment:
there is an update to the generator, which also put the requirements in `bear-requirements.yaml`. Se travis log.